src/mito2/src/memtable/time_series.rs

```diff
@@ -892,7 +893,7 @@ impl ValueBuilder {
         sequence: u64,
         op_type: u8,
         fields: impl Iterator<Item = ValueRef<'a>>,
-    ) -> usize {
```

Comment:
https://github.com/GreptimeTeam/greptimedb/blob/b53a0b86fb8b2f3cede177025c8a9ffb4bb40e3a/src/mito2/src/memtable/time_series.rs#L887-L889

It was by design to not returning an error here. @v0y4g3r  WDYT?
I'm fine to change this to return a result, but you should also update the comment.

Also we can't panic here otherwise we can't recover the write worker task.

Reply:
> https://github.com/GreptimeTeam/greptimedb/blob/b53a0b86fb8b2f3cede177025c8a9ffb4bb40e3a/src/mito2/src/memtable/time_series.rs#L887-L889
> 
> It was by design to not returning an error here. @v0y4g3r WDYT? I'm fine to change this to return a result, but you should also update the comment.

Yes, by design it's not expected to encounter an error in `Series::push`.